Controllers/TimeWorkedAndVacationController.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using stunning_robot_HR.Data;
+using stunning_robot_HR.Models;
+
+namespace stunning_robot_HR.Controllers
+{
+    public class TimeWorkedAndVacationController : Controller
+    {
+        private readonly stunning_robot_HRContext _context;
+
+        public TimeWorkedAndVacationController(stunning_robot_HRContext context)
+        {
+            _context = context;
+        }
+
+        // GET: TimeWorkedAndVacation
+        public async Task<IActionResult> Index()
+        {
+            return View(await _context.TimeWorkedAndVacation.ToListAsync());
+        }
+
+        // GET: TimeWorkedAndVacation/Details/5
+        public async Task<IActionResult> Details(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var timeWorkedAndVacation = await _context.TimeWorkedAndVacation
```

Comment:
This nice little query should like live in a repository. Is it time for the TimeWorkedAndVacation repo? With corresponding ITimeWorkedAndVacation...